stable/mcrouter/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+name: mcrouter
+home: https://github.com/facebook/mcrouter
+version: 0.1.0
+appVersion: 1.0
```

Comment:
Looks like mcrouter is at https://github.com/facebook/mcrouter/tree/v0.36.0 does this map to what your image pulls in?

Reply:
This is tricky. The third-party image that is used here doesn't clearly pin down a version: https://github.com/jamescarr/mcrouter-docker/blob/master/Dockerfile#L11

Now if you run:
```
helm install stable/mcrouter --name=myproxy

POD=$(kubectl get pods -l app=mycache-mcrouter -o jsonpath="{.items[0].metadata.name}")

kubectl exec -it $POD -- mcrouter --version
```

... you get this output:

```
mcrouter 1.0
```

... which I would agree doesn't quite make sense, given that the current version is 0.36.0.

My guess is that they've changed their versioning scheme since that image was created. I'll have to look into this more.